src/main/java/io/javalin/plugin/openapi/OpenApiHandler.kt

```diff
@@ -3,32 +3,72 @@ package io.javalin.plugin.openapi
 import io.javalin.Javalin
 import io.javalin.core.event.HandlerMetaInfo
 import io.javalin.core.util.Header
+import io.javalin.core.util.OptionalDependency
+import io.javalin.core.util.Util
 import io.javalin.http.Context
 import io.javalin.http.Handler
 import io.javalin.plugin.openapi.annotations.ContentType
 import io.javalin.plugin.openapi.annotations.OpenApi
+import io.swagger.util.Yaml
 import io.swagger.v3.oas.models.OpenAPI
+import io.swagger.v3.parser.OpenAPIV3Parser
+import org.slf4j.LoggerFactory
+
+private val logger = LoggerFactory.getLogger(OpenApiHandler::javaClass.javaClass)
```

Comment:
`javaClass.javaClass` is probably not correct

```suggestion
private val logger = LoggerFactory.getLogger(OpenApiHandler::class.java)
```